modules/dicom_archive/README.md

```diff
@@ -0,0 +1,43 @@
+# Dicom Archive
+
+## Purpose
+
+The DICOM Archive module provides a front end to view the metadata
+about DICOMs which have been processed by the LORIS database via
+the imaging pipeline. It includes both failed and successful scans.
+
+## Intended Users
+
+The DICOM Archive is primarily used by DCC staff who are investigating
+the status of imaging pipeline scripts.
+
+## Scope
+
+The DICOM Archive shows DICOM tar metadata. It does not provide a
+way to visualize scans or view metadata of other data formats. It
+provides a frontend to the `tarchive_*` tables in LORIS, but does
+not directly inspect files on the filesystem.
```

Comment:
perhaps mentioning something about the md5sum since the module displays it (in the same context that the module does not inspect the files on the filesystem but tracks changes to the .tar through tarchive_validation.pl later on).
JUst a thought but might be unnecessary if it complicates the text.

Reply:
I think that might be too specific and be distracting in this overview document.